npm/qsharp/ux/qsharp-circuit.css

```diff
@@ -6,7 +6,10 @@
   p {
     padding: 10px 0px;
   }
+}
 
+/* Circuit widget */
+.qs-circuit-widget {
```

Comment:
I'm not sure you need another layer of CSS classes here. Ultimately, everything you care about styling in this block is already in a `qs-circuit` styled DIV. A lot of the styles below are already prefixed with `.qs-circuit`. Maybe you just need to elevate `.qs-circuit` to be the outer CSS scope here.

Reply:
done